app/views/TaggingFlow/PrimaryTagging/index.tsx

```diff
@@ -23,6 +26,41 @@ function PrimaryTagging(props: Props) {
         ]));
     }, [setEntries]);
 
+    const handleEntryDelete = React.useCallback((entryId: string) => {
+        setEntries((oldEntries) => {
+            const i = oldEntries.findIndex((e) => e.clientId === entryId);
+
+            if (i === -1) {
+                return oldEntries;
+            }
+
+            const newEntries = [...oldEntries];
+            newEntries.splice(i, 1);
+
+            return newEntries;
+        });
+    }, [setEntries]);
+
+    const handleExcerptChange = React.useCallback((entryId: string, modifiedExcerpt: string) => {
+        setEntries((oldEntries) => {
+            const i = oldEntries.findIndex((e) => e.clientId === entryId);
+
+            if (i === -1) {
+                return oldEntries;
+            }
```

Comment:
Let's also add a console.error as this case should not occur.

Reply:
This logic was implemented only for demo purpose and probably will change